docs/users/configuration.md

```diff
@@ -0,0 +1,185 @@
+| `enabled`      	| Boolean 	| `false`  	| Whether or not Velocity should reply to GameSpy 4 (Minecraft query protocol) requests. You can usually leave this false. 	|
+| `port`         	| Number  	| `25577`  	| Specifies which port that Velocity should listen on for GameSpy 4 (Minecraft query protocol) requests.                   	|
+| `map`          	| String  	| Velocity 	| Specifies the map name to be shown to clients.                                                                           	|
+| `show-plugins` 	| Boolean 	| false    	| Whether or not Velocity plugins are included in the query responses.                                                     	|
+
+##`metrics` section
+
+| Setting name  	| Type    	| Default                 	| Description                                                                                                 	|
+|---------------	|---------	|-------------------------	|-------------------------------------------------------------------------------------------------------------	|
+| `enabled`     	| Boolean 	| `true`                  	| Whether or not Velocity should send metrics to bStats.                                                      	|
+| `id`          	| UUID    	| Randomly generated UUID 	| A randomly generated UUID that uniquely identifies your Velocity server. You should not alter this setting. 	|
+| `log-failure` 	| Boolean 	| `false`                 	| Whether or not Velocity should log whenever it fails to connect to bStats.                                  	|
+
+
+##The default configuration
+
+Below is the default configuration file for Velocity, `velocity.toml`.
+
+```toml
+
+# Config version. Do not change this
+config-version = "1.0"
+
+# What port should the proxy be bound to? By default, we'll bind to all addresses on port 25577.
+bind = "0.0.0.0:25577"
+
+# What should be the MOTD? This gets displayed when the player adds your server to
+# their server list. Legacy color codes and JSON are accepted.
+motd = "&3A Velocity Server"
+
+# What should we display for the maximum number of players? (Velocity does not support a cap
+# on the number of players online.)
+show-max-players = 500
+
+# Should we authenticate players with Mojang? By default, this is on.
+online-mode = true
+
+# Should we forward IP addresses and other data to backend servers?
+# Available options:
+# - "none":   No forwarding will be done. All players will appear to be connecting from the
+#             proxy and will have offline-mode UUIDs.
+# - "legacy": Forward player IPs and UUIDs in a BungeeCord-compatible format. Use this if
+#             you run servers using Minecraft 1.12 or lower.
+# - "modern": Forward player IPs and UUIDs as part of the login process using Velocity's
+#             native forwarding. Only applicable for Minecraft 1.13 or higher.
+player-info-forwarding-mode = "NONE"
+
+# If you are using modern IP forwarding, configure an unique secret here.
+forwarding-secret = "5L7eb15i6yie"
```

Comment:
Probably a good idea _NOT_ to include this (or at least put in an obvious dummy string here)

Reply:
This iteration of the config was copied from the existing docs. However I can still change it